storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -4676,7 +4875,7 @@ static bool get_meta_path(
 {
   size_t len = strlen(delta_path);
 
-  if (len <= 6 || strcmp(delta_path + len - 6, ".delta")) {
+  if (len <= 6 || strcmp(delta_path + len - 6, EXT_DELTA.c_str())) {
```

Comment:
possible to avoid 6 ? strlen(EXT_DELTA). Simlar comment to other EXT variants introduced in this code.

Reply:
AB